torchmultimodal/models/masked_auto_encoder/model.py

```diff
@@ -20,6 +20,13 @@
 )
 
 
+MAE_MODEL_MAPPING = {
```

Comment:
isnt it nicer to expose as vit_mae* wrapper with pretrained=True like clip in here itself vs making the user pass it around

Reply:
Yeah that's fine too. I thought it was a bit weird cause we are then mixing builders across files (i.e. we either have a builder for MAE inside vision_transformer.py or a builder returning ViT inside MAE model.py). But maybe the second option (I think that's what you're suggesting?) isn't so bad